src/run-command.php

```diff
@@ -54,6 +55,7 @@ function runCommand($context, $command)
         "who-works-when" => 1,
         "update-remote-service" => 2,
         "wiki-api" => 2,
+        "wiki-api-import" => 2,
```

Comment:
what's the difference between the `wiki-api` command and the `wiki-api-import` command?